tools/application_performance_profiler.py

```
@@ -29,9 +29,12 @@ def application_performance_profiler():
 
-def main():
-    application_performance_profiler()
-
-
-if __name__ == "__main__":
-    main()
+def check_platform_compatibility():
+    supported = True
+    warnings = []
+    try:
+        import psutil
```

Ruff (0.11.9)

`psutil` imported but unused

Remove unused import: `psutil`

(F401)